app.py

```diff
@@ -212,6 +212,26 @@ def check():
         return jsonify(claim_code.to_json())
     return abort(404)
 
+@app.route("/merchanttxlog", methods=["POST"])
```

Comment:
change to just merchanttx

Reply:
@app.route("/merchanttx", methods=["POST"])
def merchanttx():
    sig = request.headers.get("X-Signature")
    content = request.json
    api_key = content["api_key"]
    nonce = content["nonce"]
    token = content["token"]
    walletaddr = content["walletaddr"]
    amount = content["amount"]
    txid = content["txid"]
    direction = content["direction"]
    type = content["type"]
    res, reason, api_key = check_auth(api_key, nonce, sig, request.data)
    if not res:
        return abort(400, reason)
    merchant_tx = MerchantTx(api_key.user, walletaddr, amount, txid, direction, type)
    db.session.add(merchanttx)
    db.session.commit()
    return jsonify(merchanttx.to_json())


i dont know if this is correct. but the models.py for direction will need to be changed from "boolean" to something else. because it will not make sense to the customer what 0 or 1 is. Also how do you enter a boolean on the parser for the test_api.py. not seen that yet.